src/settingsTab/index.ts

```diff
@@ -42,6 +43,24 @@ export class SettingsTab extends PluginSettingTab {
     this.sponsorMe();
   }
 
+
+  private async myClippingsFileLocation(): void {
+    new Setting(this.containerEl)
+      .setName('My Clippings file location')
```

Comment:
Would this preference make more sense if it was called "Default My Clippings file location"?